pkg/engine/deployment.go

```diff
@@ -340,8 +339,22 @@ func (deployment *deployment) run(cancelCtx *Context, actions runActions, policy
 	duration := time.Since(start)
 	changes := actions.Changes()
 
+	// Refresh and Import do not execute Policy Packs.
+	policies := map[string]string{}
+	if !deployment.Options.isRefresh && !deployment.Options.isImport {
+		for _, p := range deployment.Options.RequiredPolicies {
+			policies[p.Name()] = p.Version()
+		}
+		for _, pack := range deployment.Options.LocalPolicyPacks {
+			//path := abbreviateFilePath(pack.Path)
```

Comment:
Local policies in the new display are rendered the same as remote policies -- notably, now with correct version numbers from the policy pack. Do we feel that they need to be rendered differently?

And good catch here, yes, will remove that!

Reply:
> Do we feel that they need to be rendered differently?

Not sure. I don't feel strongly about it either way... It seems like in the past there was some effort to be able to distinguish between a policy pack run locally vs. one enforced by Pulumi Cloud, displayed by both the CLI and in Pulumi Cloud (https://github.com/pulumi/pulumi/pull/3839, https://github.com/pulumi/pulumi-service/pull/4591)

I'm not sure how much of a need there is for this in practice, though. Kind of a stretch, but perhaps when you're developing a policy pack on a stack that also has cloud-enforced policy packs applied? To be able to distinguish during development which one is producing violations (one that's already published vs. one that you're developing locally)? Probably not something we need to be too concerned with.

Reply:
Yeah it was only shown in the summary table at the end, which is now gone in favor of the consolidated view. I am absolutely happy to bring it back. Just requires plumbing a bit more info around. This area of the codebase seemed sort of buggy so I tried to clean some of it up (e.g., it seemed to accidentally drop the version on the floor because it mutated a copy of a struct rather than the source struct actually being used).

Reply:
I went ahead and brought this back. Local packs now are marked as such, along with their path.